quickcheck/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+Unreleased
+---------------
+
+_YYYY-MM-DD_
+
+* Print quickcheck progress using tasty progress reporting.
```

Comment:
Could you please update `tasty-quickcheck.cabal` requiring `tasty >= 1.5`? Otherwise entities like `emptyProgress` are not available.

Reply:
Hi from the future, it appears that the bound on tasty in tasty-quickcheck-0.10.3 is not restrictive enough (`>=1.0.1 && <1.6`) and should really be along the lines of (`>=1.5`), because the import of `emptyProgress` is not conditioned on the version of tasty chosen by cabal.

This has been observed in CI when trying to add support for text-2.1 in Cabal and Cabal-syntax https://github.com/haskell/cabal/actions/runs/6133692735/job/16646973355?pr=9242#step:8:1423

Reply:
Thanks, fixed in https://hackage.haskell.org/package/tasty-quickcheck-0.10.3/revisions/.